yt/utilities/parameter_file_storage.py

```diff
@@ -114,7 +126,17 @@ def _convert_ds(self, ds_dict):
         fn = os.path.join(fp, bn)
         class_name = ds_dict["class_name"]
         if class_name not in output_type_registry:
-            raise UnknownDatasetType(class_name)
+            # first check that the relevant frontend is imported:
+            # the output_type_registry is refreshed between sessions as
+            # frontends are imported on-demand now.
+            import importlib
```

Comment:
Any reason not to put this import at the top level ? importlib should be "free" to import anyway.